internal/util/debug/debug.go

```diff
@@ -86,7 +87,9 @@ func Listen(opts *ListenOpts) (*Handler, error) {
 
 	must.NotBeZero(opts)
 
-	stdL := must.NotFail(zap.NewStdLogAt(opts.L, zap.WarnLevel))
+	l := opts.L
+
+	stdL := slog.NewLogLogger(l.Handler(), slog.LevelError)
```

Comment:
@noisersup should this use `Warn` or `Error`?

Reply:
@noisersup

Reply:
The response was already on slack (error)